Source/SLKTextViewController.m

```diff
@@ -892,6 +906,10 @@ - (void)setTextInputbarHidden:(BOOL)hidden animated:(BOOL)animated
     }
     
     _textInputbar.hidden = hidden;
+
+    if (@available(iOS 11.0, *)) {
+        [self viewSafeAreaInsetsDidChange];
+    }
```

Comment:
I think this block should be in animation block when constraint updated right below